src/constants/ssm-parameter-paths.ts

```diff
@@ -1,6 +1,7 @@
 export interface SsmParameterPath {
   path: string;
   description: string;
+  optional?: boolean;
```

Comment:
The `account-readiness` CLI command currently expects all these parameters to be required, but actually we're introducing an optional, account-wide parameter here (account-wide because you might have multiple instances of the fastly IAM logging construct in a single account)